ui/src/Utils/helpers.js

```diff
@@ -19,19 +21,15 @@ export function compareValues(key, order = TABLE.ASCENDING) {
       return 0;
     }
 
-    const varA = (typeof a[key] === 'string')
-      ? a[key].toUpperCase() : a[key];
-    const varB = (typeof b[key] === 'string')
-      ? b[key].toUpperCase() : b[key];
+    const varA = typeof a[key] === "string" ? a[key].toUpperCase() : a[key];
```

Comment:
Can we have better naming for variable varA and varB doesn't describe it's purpose

Reply:
Hi @DeRaowl this is a comparator function. What do you think would be better? Since those are keys can i name them key1 and key2?

Reply:
Ok instead you can just add small comment saying what these variables are